backups/snapshots/2025-08-30_15-46-05/scraper.py

```
@@ -0,0 +1,345 @@
+                except Exception as exc:  # noqa: BLE001
+                    self.logger.debug(
+                        f"No se pudo procesar el payload JSON de la API {response.url}: {exc}"
+                    )
+                    return
+                payload_str = json.dumps(json_payload, sort_keys=True)
+                payload_hash = hashlib.sha256(payload_str.encode("utf-8")).hexdigest()
+                try:
+                    self.db_manager.save_discovered_api(
+                        page_url=self.page.url, api_url=response.url, payload_hash=payload_hash
+                    )
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)